docs/guides/nxp_k32w_android_commissioning.md

```diff
@@ -3,7 +3,7 @@
 This article describes how to use
 [CHIPTool](../../src/android/CHIPTool/README.md) for Android smartphones to
 commission an NXP K32W061 DK6 running
-[NXP K32W Lock/Light Example Application](../../examples/lock-light-app/k32w/README.md)
+[NXP K32W Lock/Light Example Application](../../examples/lighting-app/k32w/README.md)
```

Comment:
Well, this should have links to both the lighting app and the lock app, right?

Reply:
You're right, I'll split the links to reflect that